flocker/node/script.py

```diff
@@ -119,6 +67,11 @@ class DatasetAgentOptions(_AgentOptions):
 
     synopsis = _AgentOptions.synopsis.format("flocker-dataset-agent")
 
+    optParameters = [
+        ["config-file", "c", FilePath("/etc/flocker/dataset-agent.yml"),
```

Comment:
Using `FilePath` as a default value unfortunately makes the help output look dumb.